### PR TITLE
Technical SEO audit line 9 - 3XX redirect

### PR DESCRIPTION
**Pull request by madebyguie**
Technical SEO audit - line 9 of document - redirects

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /docs/basic-config-and-security/security/authentication-authorization/oauth2.0/auth-code-grant/ /docs/basic-config-and-security/security/authentication-&-authorization/oauth2-0/auth-code-grant/
-/docs/basic-config-and-security/security/authentication-authorization/oauth2.0/refresh-token-grant/ /docs/basic-config-and-security/security/authentication-authorization/oauth2-0/refresh-token-grant
```
Comment (letzya) [inline _redirects:2]: This file is auto-gen. need to remove it from the PR

File: tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ This plugin allows you to manually select each endpoint for tracking.
 
 ### URL Rewrite
 
-This plugin allows you to translate an outbound API interface to your internal structure of your services. See [URL Rewriting](/docs/advanced-configuration/transform-traffic/url-rewriting/) for more details.
+This plugin allows you to translate an outbound API interface to your internal structure of your services. See [URL Rewriting](/docs/transform-traffic/url-rewriting/) for more details.
```
Comment (letzya) [inline tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md:166]: Need to update it to use `{{< ref "" }}`

File: tyk-docs/content/api-management/api-management.md [hunk 3]
```diff
@@ -69,4 +69,4 @@ Head on over to the [OSS section][11] for more information on it and the other o
 With Tyk Cloud all of the above closed-source components are available. Get your free account [here][13].
 
 
-There are many open and closed source [Tyk components](/docs/tyk-stack) that make up the various solutions.
+There are many open and closed source [Tyk components](/docs/tyk-stack/) that make up the various solutions.
```
Comment (letzya) [inline tyk-docs/content/api-management/api-management.md:72]: change to `{{< ref "" >}}`

File: tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/json-web-tokens.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ We are telling Tyk to extract this unique ID from the `sub` Header, which is the
 
 #### Step 4: Set a Default Policy
 
-If Tyk cannot find a `pol` claim, it will apply this Default Policy. Select a policy that gives access to this API we are protecting, or [go create one first](/docs/getting-started/tutorials/create-security-policy/) if it doesn't exist.
+If Tyk cannot find a `pol` claim, it will apply this Default Policy. Select a policy that gives access to this API we are protecting, or [go create one first](/docs/getting-started/create-security-policy/) if it doesn't exist.
```
Comment (letzya) [inline tyk-docs/content/basic-config-and-security/security/Authentication & Authorization/json-web-tokens.md:44]: change link to {{< ref "" >}}

**Review by letzya — CHANGES_REQUESTED** [review]
conflicts to resolve @itsmartins

**Comment by madebyguie** [comment]
Redundant PR

**Pull request closed without merging**